app/views/classified_listings/_form.html.erb

```diff
@@ -78,7 +78,7 @@
         <h3>Listings Require Credits</h3>
       <% end %>
       <div class="listings-current-credits-inner">
-        You have <%= pluralize @credits.size, "credit" %> available — <a href="/credits/purchase" data-no-instant>Buy More</a>
+        You have <%= pluralize @credits.size, "credit" %> available — <a target="_blank" href="/credits/purchase" data-no-instant>Buy More</a>
```

Comment:
Can you add `rel="noopener noreferrer"` to the `<a>` tags with `_blank`?

The reason is this curious vulnerability: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

👀